tests/gis_tests/layermap/tests.py

```diff
@@ -324,6 +325,20 @@ def allow_relation(self, obj1, obj2, **hints):
     def allow_migrate(self, db, app_label, **hints):
         return True
 
+    def test_null_geom_with_unique(self):
```

Comment:
The test won't run if it's in the database router.

Reply:
No idea why it ended up there, apologies. Can I blame patch?

Reply:
I'm not sure what you mean.